sauce.browsers.js

```diff
@@ -0,0 +1,141 @@
+const sauceBrowsers = {
```

Comment:
We have webpack.config in a separate file, why not for the saucelab?
Plus, it's really long, and I think it just make sense to put this in a separate file.